src/http_api/api_types.rs

```diff
@@ -106,3 +106,26 @@ pub struct LedgerBalanceResponse {
 pub struct GnosisSafeConfigResponse {
     pub config: GnosisSafeConfig,
 }
+
+/// Mint with Config
+#[derive(Serialize, Debug, Eq, PartialEq)]
+#[allow(missing_docs)]
+pub struct MintWithConfig {
+    pub mint_tx: MintTx,
+    pub mint_config: MintConfig,
```

Comment:
I'm not sure what information you are displaying in the block explorer, but if you want to go from a MintTx all the way to the block that contains the matching MintConfigTx, you'll also need to include the MintConfigTx here that contains the MintConfig